CHANGES.md

```diff
@@ -15,7 +15,12 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+- Code documentation: extra reason to allow not existing dir in `EXTRA_CONF_DIRS`
```

Comment:
Can we change this to:

- Code documentation: provide an additional reason to not exit early if a directory listed in the `EXTRA_CONF_DIRS` variable does not exist.

(or similar)

Reply:
Thanks, updated.  English is not my native language!